From 2b20c5b4f713a3eff73730c4e421ab025c9d59c8 Mon Sep 17 00:00:00 2001
From: "P. Sadik" <psadik@broadcom.com>
Date: Mon, 13 Aug 2012 19:32:54 +0530
Subject: [PATCH 588/761] Fix: Disable native SATA Driver. The native SATA
 Driver is not yet ported. Moreover, enabling this
 option breaks compilation.

Based on Broadcom SDK 2.3.

Signed-off-by: P. Sadik <psadik@broadcom.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/ata/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ata/Makefile b/drivers/ata/Makefile
index 52af4ab..13286b5 100644
--- a/drivers/ata/Makefile
+++ b/drivers/ata/Makefile
@@ -3,7 +3,7 @@ obj-$(CONFIG_ATA)		+= libata.o
 
 # non-SFF interface
 obj-$(CONFIG_SATA_AHCI)		+= ahci.o libahci.o
-obj-$(CONFIG_SATA_AHCI)		+= xlp-ahci.o
+#obj-$(CONFIG_SATA_AHCI)		+= xlp-ahci.o
 obj-$(CONFIG_SATA_ACARD_AHCI)	+= acard-ahci.o libahci.o
 obj-$(CONFIG_SATA_AHCI_PLATFORM) += ahci_platform.o libahci.o
 obj-$(CONFIG_SATA_FSL)		+= sata_fsl.o
-- 
1.7.10.4

